src/main/java/tetris/Tick.java

```diff
@@ -86,4 +89,9 @@ public long getTime() {
     public void setTime(long time) {
         this.time = time;
     }
+
+    @Override
+    public void update(Observable o, Object arg) {
+        this.time = (long) Math.max(1, 200 * Math.exp(-.0005 * (int) arg));
```

Comment:
(int) arg = score? Clearing 3 lines is decreasing the tick interval from 200ms to approx. 172ms. Personally, I think clearing 3 lines shouldn't already increase the game speed.